pyhomematic/devicetypes/sensors.py

```diff
@@ -209,8 +209,20 @@ def is_motion_detection_active(self, channel=None):
         return bool(self.getBinaryData("MOTION_DETECTION_ACTIVE", channel))
 
     def get_brightness(self, channel=None):
-        """ Return brightness from 0 (dark ) to 255 (bright) """
-        return int(self.getSensorData("ILLUMINATION", channel))
+        """ Return brightness from 0 (dark ) to 163830 (bright) """
```

Comment:
163830 as the upper boundary seems incredibly high. We just take the value that gets submitted, so it's not really an issue. Just want to make sure this is the correct value.
And there's a whitespace charater after "dark" and after the ) from bright. Might as well remove that. After that's done you can squash.